src/components/Timeline/TextEditor/helpers.ts

```diff
@@ -0,0 +1,223 @@
+import isUrl from 'is-url';
+import {
+  Descendant,
+  Editor,
+  Range,
+  Element as SlateElement,
+  Transforms,
+} from 'slate';
+import isHotkey, { isKeyHotkey } from 'is-hotkey';
+
+const LIST_TYPES = ['numbered-list', 'bulleted-list'];
+const TEXT_ALIGN_TYPES = ['left', 'center', 'right', 'justify'];
+const HOTKEYS: { [key: string]: string } = {
+  'mod+b': 'bold',
+  'mod+i': 'italic',
+  'mod+s': 'strikethrough',
```

Comment:
Mod+S are often "save", isn't it? Google Docs and some other misc tools I use have "mod+x" as the strikethrough hotkey. I think that's a better option. What do you think?